src/test/online-boutique/values.yaml

```diff
@@ -0,0 +1,66 @@
+ad:
+  name: adservice
+  version: v1
+  image:
+    repository: registry.cn-beijing.aliyuncs.com/kubesphereio/adservice:v0.2.3
```

Comment:
Please consider using a flex way to describe the images. The target users are global instead of Chinese only. It's quite strange for the overseas users when they see the domain `registry.cn-beijing.aliyuncs.com`.

Reply:
Yeah. Use gcr.io by default, aliyun-images can be used alternative.